spec/requests/api/offers/update_spec.rb

```diff
@@ -24,6 +24,11 @@
       it 'responds offer message' do
         expect(response_json['message']).to eq "You accepted help from #{helper.email}"
       end
+
+      it 'sets the status of the request to "active"' do
+        request.reload
+        expect(request.status).to eq "active"
+      end
```

Comment:
below in that file(nothing you have touched so cant directly comment on the line) one of the it block says 'has a 422 response' but then expect status 500. Doesn't affect anything but I thought we should correct it!